lib/utils/helpers.ts

```diff
@@ -59,19 +60,37 @@ export function stateToString(state: OperationState) {
 	}
 }
 
-export function sudo(
+/**
+ * Execute a CLI command with admin / superuser privileges, prompting
+ * the user as needed (e.g. for a sudo password on Linux/Mac, or for
+ * Yes/No GUI prompt on Windows).
+ * @param command A CLI command without the 'node' or 'balena' prefixes,
+ * for example: ['os', 'initialize', ...]
+ * @param param1.msg Optional message for the user, before the password prompt
+ * @param param1.stderr Optional stream to which stderr should be piped
+ */
+export async function sudo(
 	command: string[],
 	{ stderr, msg }: { stderr?: NodeJS.WritableStream; msg?: string } = {},
 ) {
-	const { executeWithPrivileges } = require('./sudo');
+	const { executeWithPrivileges } = await import('./sudo');
+
+	// In the case of a CLI standalone zip package (process.pkg is truthy),
+	// the Node executable is bundled with the source code and node_modules
+	// folder in a single file named in argv[0]. In this case, argv[1]
+	// contains a "/snapshot" path that should be ignored.
+	const fullCommand = (process as any).pkg
+		? [process.argv[0], ...command]
+		: [process.argv[0], process.argv[1], ...command];
 
 	if (os.platform() !== 'win32') {
 		console.log(
-			msg || 'If asked please type your computer password to continue',
+			msg ||
+				'Admin privileges required: you may be asked for your computer password to continue.',
```

Comment:
```suggestion
				'Admin privileges required: you may be asked for your password to continue.',
```

Reply:
Hmm "computer password" was already there before. I actually considered it and I think it's helpful, specially for beginners, because `sudo` is being executed behind the scenes (users may not know it) and one might think that "your password" is the balenaCloud password, or some other password. I also thought of "login password", but login to what service? "Computer password" is actually somewhat clearer, I think.